keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -68,8 +67,7 @@ class ClusterHandler(
 
   override suspend fun toResolvedType(resource: Resource<ClusterSpec>): Map<String, ServerGroup> =
     with(resource.spec) {
-      val resolvedImages = imageResolver.resolveImageId(resource)
-      resolve(resolvedImages).byRegion()
+      resolve().byRegion()
```

Comment:
`ClusterHandler` no longer treats `ImageResolver` as a special case - it's just another resolver.